content/ru/horizontal-scaling.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Горизонтальное масштабирование
+status: Completed
+category: Concept
+tags: ["infrastructure", "", ""]
+---
+
+Горизонтальное масштабирование — это метод, при котором производительность системы увеличивается за счет добавления [узлов](/nodes/). 
+Этим оно отличается от [вертикального масштабирования](/vertical-scaling/), когда ресурсы добавляются на сами узлы. 
+Допустим, у нас есть система с 4 ГБ ОЗУ и нужно увеличить ее емкость до 16 ГБ ОЗУ. 
+При горизонтальном масштабировании мы добавим еще три инстанса (узла) с 4 ГБ ОЗУ, при вертикальном увеличим объем ОЗУ до 16 ГБ на имеющемся инстансе.
```

Comment:
This might be a matter of personal preference, but seems to be a bit closer to the original wording:
```suggestion
При горизонтальном масштабировании мы добавим еще три инстанса (узла) с 4 ГБ ОЗУ вместо того, чтобы увеличивать объем ОЗУ до 16 ГБ на единственном имеющемся инстансе.
```